src/Component.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+
+export class KeyPressed extends Component {
+  state = {
+    keyPressed: '',
+  };
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ keyPressed: 'Nothing was pressed' });
```

Comment:
you shouldn't do it in the `componentDidMount` with a timeout
actually, you shouldn't do it at all ;)

Reply:
should be `Nothing was pressed yet` (to pass tests)